ush/config.sh.aws_cloud

```diff
@@ -0,0 +1,58 @@
+ACCOUNT=zrtrr
+
+MACHINE=LINUX
+WORKFLOW_MANAGER=rocoto
+SCHED=slurm
+
+NCORES_PER_NODE=40
+PARTITION_DEFAULT="hera"
+QUEUE_DEFAULT=batch
+PARTITION_HPSS=service
+QUEUE_HPSS=batch
+PARTITION_FCST=hera
+QUEUE_FCST=batch
```

Comment:
Yes. The goal of this PR is to get the Linux target working on Hera. Isolating these changes should also help @mkavulich and others at DTC with implementing Rocoto support for generic Linux environments. The next step for us will be to make the changes needed to run this on AWS instead of Hera.